### ISSUE TITLE
NumberOfSpaces option does not work

### ISSUE THREAD

**Issue by Kline-** [pendo324/EssentialNowPlaying#27]
Looks like it may have been accidentally removed in the last merge? Using the 2.0.5 release posted here.

**Comment by tdt11** [comment]
Yes, this feature does not seem to work in this version or the latest release 2.0.6. This would be an amazing addition.
